server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -23,6 +26,27 @@
     </div>
   </div>
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form id="form-select-characteristic" method="get" novalidate>
+        <div class="govuk-form-group">
+          <label class="govuk-label" for="sort">
+            Select a protected characteristic
+          </label>
+          <select class="govuk-select" id="characteristic" name="characteristic">
+              {%- for option in characteristicOptions -%}
+                <option value="{{ option.value }}" {%- if option.selected -%}selected{%- endif -%}>{{ option.label }}</option>
+              {%- endfor %}
+          </select>
+
+          <button class="govuk-button govuk-!-margin-left-2" data-module="govuk-button">
```

Comment:
i'm on the fence about preventing double-clicks… the page is not exactly quick to load. happy to leave it for now :)

Reply:
Yeah, when we were working on this we didn't add it because it seemed overkill given it's a `GET` request.
But tbh it doesn't harm to have it so we're adding 👌